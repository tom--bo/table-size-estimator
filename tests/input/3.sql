create table t2 (
  id int not null primary,
  c0 date not null,
  c1 timestamp(6) not null,
  c2 bigint(20) unique key,
  c3 decimal(12,18),
  c4 varchar(32)
);
