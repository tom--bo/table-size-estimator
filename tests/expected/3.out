Input Table Definition: (Please type ^d to end input) 
successfully ended
------
1 row max size = 179 bytes .
1 row Average size = 114 bytes .
